dashboard/app/models/levels/level.rb

```diff
@@ -844,6 +846,7 @@ def summarize_for_lab2_properties(script, script_level = nil, current_user = nil
     properties_camelized[:appName] = game&.app
     properties_camelized[:useRestrictedSongs] = game.use_restricted_songs?
     properties_camelized[:usesProjects] = try(:is_project_level) || channel_backed?
+    properties_camelized[:finishUrl] = script_completion_redirect(current_user, script) if script
```

Comment:
Should we be setting `finishDialog` here as well? Or is that done somewhere else?

Reply:
We get it for free when it's set on `properties` in the level.

Reply:
Oh convenient!